startup_assistance/indico_startup_assistance/models/startup_assistance_requests.py

```diff
@@ -0,0 +1,38 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from indico.core.db.sqlalchemy import db
+from indico.util.string import return_ascii
+
+
+class StartupAssistanceRequest(db.Model):
+    """Startup assistance requests"""
+
+    __tablename__ = 'startup_assistance_requests'
+    __table_args__ = {'schema': 'plugin_startup_assistance'}
+
+    reservation_id = db.Column(
+        db.ForeignKey('roombooking.reservations.id'),
+        primary_key=True
+    )
+
+    registration = db.relationship(
+        'Reservation',
+        uselist=False,
+        lazy=True,
```

Comment:
you probably want `lazy=False` here  so you get the registrations when fetching the requests for the list of active assistance requests